src/Service/Models/GraphQLFilterParsers.cs

```diff
@@ -542,7 +542,7 @@ public static class FieldFilterParser
                 predicates.Push(new PredicateOperand(new Predicate(
                     new PredicateOperand(column),
                     op,
-                    new PredicateOperand(processLiteral ? $"{processLiterals(value)}" : value.ToString()))
+                    new PredicateOperand(processLiteral ? $"{processLiterals(value, column.ColumnName)}" : value.ToString()))
```

Comment:
>processLiterals(value, column.ColumnName) [](http://example.com/codeflow?start=61&length=41)

Does this not take three values now as indicated by your changes earlier in this file? `Func<object, string?, string> processLiterals`

Reply:
Func<object,string?> -> Represents function accepts object,string? as arguments.
string -> Represents function returns string.